### PR TITLE
Resolve issue #73 about the TrackException method

### PR DESCRIPTION
**Pull request by gzepeda**
- Changed the definition of the trackException method to have the additional parameter called measurements.
- Changed the definition of the getExceptionData method to consider the measurements parameter
- No need to change the Exception Data Contract since it already had a measurements property

### CODE REVIEW

File: Tests/Library/Client.tests.ts [hunk 1]
```diff
@@ -124,13 +124,20 @@ describe("Library/Client", () => {
...
-            assert.ok(trackStub.calledTwice);
+            assert.ok(trackStub.calledThrice);
 
             var args = trackStub.args;
+            
             assert.equal(args[0][0].baseData.exceptions[0].message, name);
+
             assert.equal(args[1][0].baseData.exceptions[0].message, name);
             assert.deepEqual(args[1][0].baseData.properties, properties);
+
```
Comment (dnduffy) [inline Tests/Library/Client.tests.ts]: Should these tests also check that the arguments which were not provided are indeed absent?

Comment (gzepeda) [inline Tests/Library/Client.tests.ts]: I believe it is a good idea, but probably for another test or in the same test?

Comment (kamilszostak) [inline Tests/Library/Client.tests.ts]: I would add a new test to cover both : missing properties and missing measurements. Thanks!

**Comment by kamilszostak** [comment]
I changed the [release process](https://github.com/Microsoft/ApplicationInsights-node.js#branches) and all new features should go to a [develop](https://github.com/Microsoft/ApplicationInsights-node.js/tree/develop) branch. 

Could you please send a pull request to develop branch? 

Thanks!

**Comment by gzepeda** [comment]
Closing to open a new one for the Develop Branch.

**Pull request closed without merging**